inc/class-sentry.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * @author  Pressbooks <code@pressbooks.com>
+ * @license GPLv3 (or any later version)
+ */
+
+namespace Pressbooks;
+
+use function \Pressbooks\Utility\debug_error_log;
+use PressbooksMix\Assets;
+
+class Sentry {
+
+	const DEFAULT_ENVIRNOMENT = 'staging';
+
+	const WP_SCRIPT_NAME = 'script-sentry';
+
+	/**
+	 * @var Sentry
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Sentry DSN
+	 *
+	 * @var string
+	 */
+	protected static $dsn;
+
+	/**
+	 * @var \WP_User
+	 */
+	private $user;
+
+	/**
+	 * @since 5.5.3
+	 *
+	 * @return Sentry
+	 */
+	static public function init() {
+
+		if ( self::areEnvironmentAvailable() ) {
+			self::$dsn = env( 'SENTRY_DSN' );
+		}
+
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+			self::$instance->getCurrentUserForTracking();
+			if (
+				! is_null( env( 'SENTRY_INITIALIZE_PHP' ) ) &&
+				intval( env( 'SENTRY_INITIALIZE_PHP' ) ) === 1
+			) {
+				self::$instance->phpObserver();
+			}
+			if (
+				! is_null( env( 'SENTRY_INITIALIZE_JAVASCRIPT' ) ) &&
+				intval( env( 'SENTRY_INITIALIZE_JAVASCRIPT' ) ) === 1
+			) {
+				self::$instance->javascriptObserver();
+			}
+		}
+
+		return self::$instance;
+
+	}
+
+	/**
+	 * Get current user for tracking (if available)
+	 *
+	 * @return false|\WP_User
+	 */
+	public function getCurrentUserForTracking() {
+		$user_id = get_current_user_id();
+		$this->user = $user_id > 0 ? $user = get_userdata( $user_id ) : false;
```

Comment:
may reduce to 
`$this->user = $user_id > 0 ? get_userdata( $user_id ) : false;`

Reply:
Yes! I will change it. It was an typing error, doesn't make sense how I did it, probably a typing error. Thank you for catching it!